Content.Shared/_Goobstation/Heretic/Heretic.Abilites.cs

```diff
@@ -24,6 +24,10 @@ public sealed partial class HereticActionComponent : Component
 {
     public bool MagicItemActive = false;
 }
+
+[Serializable, NetSerializable] public sealed partial class EldritchItemDoAfterEvent : SimpleDoAfterEvent
+{
+}
```

Comment:
a little spacing under this for me? since this body is empty, you can also just end line 28 with `;` and it'll take up a little less space